lib/gui/models/store.js

```diff
@@ -91,7 +105,8 @@ const storeReducer = (state, action) => {
 
       const newState = state.set('availableDrives', Immutable.fromJS(action.data));
```

Comment:
>  We could reduce the chances of typos by making constants but I wonder if it's too much complexity

\*shrug\*

> trying to mitigate the fact that JavaScript is not a strongly typed language

Even a strongly-typed language doesn't prevent making typos in strings-used-as-constants ;-P

> Maybe moving to TypeScript is the solution?

I'm still getting to grips with ES, I'm in no hurry to switch to TS ;-D